packages/fast-components-react-msft/src/divider/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as React from "react";
+import { Divider, IDividerClassNameContract, IDividerHandledProps, IFoundationProps } from "@microsoft/fast-components-react-base";
```

Comment:
can you pull the spaces between the brackets and the text per our most current conversation for consistency?